spynnaker/pyNN/models/pynn_population_common.py

```diff
@@ -19,6 +19,10 @@
 
 
 class PyNNPopulationCommon(object):
+
+    # Keep a global neuron id for print headers
+    _next_global_neuron_id = 0
```

Comment:
This appears to be unreferenced elsewhere.

Reply:
Oops - should probably go now...